README.md

```diff
@@ -50,14 +50,49 @@ Common Operational Datasets (CODs) on HDX.
 
 ## Reproducing this analysis
 
-Create a new virtual environment and install the requirements with
+Create a directory where you would like the data to be stored,
+and point to it using an environment variable called
+`AA_DATA_DIR`.
+
+Next create a new virtual environment and install the requirements with:
 
 ```shell
-pip install -r requirements.txt
 pip install --index-url https://test.pypi.org/simple/ \
 --extra-index-url https://pypi.org/simple aa-toolbox==0.4.0.dev9
+pip install -r requirements.txt
```

Comment:
Hmm I did that order cause if not installing `chardet` before toolbox on Mac, I was getting an error..

Reply:
Yeah so sorry, you can switch it back. I had some problem with conflicting packages but actually you're right and they can be switched back.